docs/development/getting-started-locally.md

```diff
@@ -131,7 +131,7 @@ To check the etcd member pods, do the following and look out for pods starting w
 kubectl get pods
 ```
 
-#### Verify Etcd Pods' Functionality
+#### Verify Etcd Pod's Functionality
```

Comment:
You can continue using `Pods'`, since the section talks about verifying multiple pods' functionality.

Reply:
Yeah makes sense.